package.json

```diff
@@ -8,6 +8,12 @@
 	"description": "Is this value a JS Date object? This module works cross-realm/iframe, and despite ES6 @@toStringTag.",
 	"license": "MIT",
 	"main": "index.js",
+	"files": [
+		"index.js",
+		"README.md",
+		"CHANGELOG.md",
+		"LICENCE"
+	],
```

Comment:
Absolutely not. The "files" array is dangerous. The only safe way to exclude files from the published package is `.npmignore`.